### ISSUE TITLE
Light/dark theme switch button

### ISSUE THREAD

**Issue by redmanbravemm** [boringprotocol/websitev2#6]
The `navbar__button` that does the theme switch should be an actual HTML `button`, for A11Y purposes and not only.

**Comment by franswan** [comment]
@redmanbravemm 😎
